### PR TITLE
Bump version to 0.1.1

### PR DESCRIPTION
**Pull request by Hkly**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Change Log
+==========
+
+# 0.1.1
+
+* Updated dependencies ([4][4])
+
+[4]: https://github.com/yola/yosps/pull/4
```
Comment (stefanor) [inline CHANGELOG.md]: That link is not really necessary. I'd leave them out.

Comment (Hkly) [inline CHANGELOG.md]: I think it sets a good example for linking future changes. Or are you saying links to pulls in general are not necessary?

**Comment by DevShep** [comment]
~tilde

File: package.json [hunk 2]
```diff
@@ -22,17 +22,17 @@
...
+    "phantomjs-prebuilt": "~2.1.4",
+    "sinon": "~1.17.3",
+    "sinon-chai": "~2.8.0"
   },
   "dependencies": {
-    "lodash": "^4.3.0",
-    "q": "^1.3.0",
-    "superagent": "1.2.0"
+    "lodash": "~4.3.0",
+    "q": "~1.3.0",
```
Comment (DevShep) [inline package.json]: Not the latest version. Same for superagent

**Comment by DevShep** [comment]
:+1:

**Pull request merged**